src/Lorentz/Contracts/BaseDAO.hs

```diff
@@ -47,8 +59,10 @@ baseDaoContract config@Config{..} = defaultContract $ contractName cDaoName $ do
     , #cFlush /-> flush config
     )
 
-fa2Handler :: (IsoValue pm) => Entrypoint FA2.Parameter (BaseDAO.Storage pm)
-fa2Handler =
+fa2Handler :: forall pm. (IsoValue pm, TypeHasDoc pm) => Entrypoint FA2.Parameter (BaseDAO.Storage pm)
+fa2Handler = do
+  docStorage @(BaseDAO.Storage pm)
+  doc $ DDescription callFA2Doc
```

Comment:
This should not be here, you should put it to `instance DocItem (DEntrypoint ...)`, otherwise it will appear dangling in the documentation.

Reply:
Can you explain?

Reply:
Well, speaking short, there should be no descriptions outside of atomic entrypoints (and fa2Handler is not an atomic entrypoints, it is a group of entrypoints). Put `callFA2Doc` into `docItemToMarkdown` of `instance DocItem (DEntrypoint FA2EntrypointKind)` instead.

The problem here is that, as you can see in the resulting documentation, we have a section with all top-level entrypoints, and a separate section with FA2 entrypoints. Unlike in code, in the resulting document FA2 entrypoints are not included into `fa2Handler` top-level entrypoint description because markdown does not work well with too deeply nested pieces.

Reply:
So I have updated instance of `FA2EntrypointKind` with callFA2Doc, but it fails the test, saying that this entrypoint is not documented:   `Entrypoint 'Call_FA2' does not contain any description.`

Reply:
Oh hmm, maybe I was wrong :thinking: 

Sorry for this. Could you when check that if you add documentation here as well then the result looks ok, and we will decide then what to do?

Reply:
I added callFA2Doc into fa2Handler and it seems to work properly now

Reply:
Nice :+1: 

Sorry again for the confusion, I really forgot how to work with this autodoc.